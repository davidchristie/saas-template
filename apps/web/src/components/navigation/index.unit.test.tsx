import { render } from "@testing-library/react";
import { Navigation } from ".";

describe("<Navigation />", () => {
  it("matches snapshot", () => {
    const result = render(<Navigation />);
    expect(result.container.firstChild).toMatchSnapshot();
  });
});
