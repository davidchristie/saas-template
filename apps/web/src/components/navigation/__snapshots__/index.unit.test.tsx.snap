// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navigation /> matches snapshot 1`] = `
<div>
  <header>
    <pre>
      null
    </pre>
    <button>
      Logout
    </button>
  </header>
  <main />
</div>
`;
